### DISCUSSION TITLE
Try to install 2 1.22.1 does not work HTTP ERROR 500

### DISCUSSION THREAD

**Discussion by Patrick-Hor** [kimai/kimai#3459] [On-Premise: Installation & Updates]
if i copy 1.9 Ayumi install in these directory it works, also 1.10 and 1.13. i also notice in downloads there are 3 assets, higher versions have 2 assets, is this the reason?

if i delete everything and copy files from 1.22 in these directory i become an HTTP ERROR 500 

What i do wrong? 

in var/log is no logfile... do i download the false version?

shared server / Linux

**Comment by ghost** [comment]
Hello,

it's possibly a permission issue. Re-assign the entire kimai installation folder back to the accessing web user.

`$ sudo chown -R username:group kimai2`
